Add .github/agents.md repository guide for AI coding agents (#1541)
HangarMC/Hangar: .github/agents.md

# Hangar — Repository Guide for AI Agents

## Project Overview

> Hangar is a **plugin repository platform** for Paper, Velocity, and Waterfall Minecraft server plugins. It is the community-maintained successor to [Ore](https://github.com/SpongePowered/Ore), rebuilt from scratch with modern technologies.

Reviewer: ```suggestion
Hangar is a **plugin repository platform** for Paper, Velocity, and Waterfall Minecraft server plugins. It is inspired by [Ore](https://github.com/SpongePowered/Ore), rebuilt from scratch with modern technologies, for PaperMC's needs.
```
